Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM golang:latest
 
-ARG HEIMDALL_DIR=/heimdall
+ARG HEIMDALL_DIR=/var/lib/heimdall
 ENV HEIMDALL_DIR=$HEIMDALL_DIR
 
 RUN apt-get update -y && apt-get upgrade -y \
     && apt install build-essential git -y \
-    && mkdir -p /heimdall
+    && mkdir -p $HEIMDALL_DIR
 
-WORKDIR ${HEIMDALL_DIR}
+WORKDIR /heimdall
```

Comment:
shouldn't work dir be ${HEIMDALL_DIR}?

Reply:
No, `HEIMDALL_DIR` will contain heimdall config and data, while `/heimdall` will only contain heimdall code.